.github/workflows/test-runner-nightly-perf.yml

```diff
@@ -49,4 +48,19 @@ jobs:
         run: |
           ./gradlew :workspace-manager-client:build
           cd workspace-manager-clienttests
+          echo "Running test suite"
           ./gradlew runTest --args="suites/BasicPerf.json /tmp/TestRunnerResults"
```

Comment:
Same comment as above: I think it's helpful to always override the server specification with the env var, so it's clear what server we're running against and so that the behavior doesn't change unexpectedly if someone changes the default server in the test suite file.

Reply:
All set.